scene/main/node.cpp

```diff
@@ -841,15 +845,19 @@ void Node::set_process_priority(int p_priority) {
 	data.process_priority = p_priority;
 
 	if (is_processing())
+		ERR_FAIL_COND(!data.tree);
```

Comment:
You need braces here and below, otherwise only the first line after the "if" is part of the if-clause.

Reply:
Oh, I missed that point. Fixed.

Reply:
Checking once before the first `if` should be enough.